web/.env.example

```
@@ -1,2 +1,25 @@
+# ---------------------------------------------------------------------------
+NEXT_PUBLIC_POSTHOG_TOKEN=
+NEXT_PUBLIC_POSTHOG_HOST=https://us.i.posthog.com
+NEXT_PUBLIC_POSTHOG_DEBUG=false #intentionally disabled for time being
+# ---------------------------------------------------------------------------
+# Playground — Vercel server-side only (no NEXT_PUBLIC_ prefix)
+# Set in Vercel → Settings → Environment Variables.
+# Read by Vercel Route Handlers. Never returned to the browser.
+# ---------------------------------------------------------------------------
+# TTS
+PLAYGROUND_ELEVENLABS_API_KEY=
+PLAYGROUND_CARTESIA_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PLAYGROUND_CARTESIA_API_KEY key should go before the PLAYGROUND_ELEVENLABS_API_KEY key

(UnorderedKey)